### PR TITLE
Basic L2 Port Programming with OVS

### PR DESCRIPTION
**Pull request by er1cthe0ne**
The current change includes:

- adopt the latest alcor schema
- add ACA_OVS_Config class for OVS setup and configuration
- updated aca_test to test ACA_OVS_Config class
- confirmed all existing and new aca_test are passing

### CODE REVIEW

**Comment by xieus** [comment]
Some path issues in concurrent commit. 

Logs: https://jenkins.alkaidcloud.io/job/alcor-control-agent-pr-test/107/console

"CMake Error at src/CMakeLists.txt:40 (add_library):
  Cannot find source file:

    ./dp_abstraction/aca_dataplane_ovs.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx"

**Comment by xieus** [comment]
The CI workflow is not trigger by the change.

**Comment by er1cthe0ne** [comment]
> The CI workflow is not trigger by the change.

ACA CI workflow is now triggered by PR also. The build filename issue has been fixed.

File: include/aca_goal_state_handler.h [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+class Aca_Goal_State_Handler {
+  public:
+  static Aca_Goal_State_Handler &get_instance();
+
+  // process ONE VPC state
+  int update_vpc_state_workitem(const alcor::schema::VpcState current_VpcState,
+                                alcor::schema::GoalStateOperationReply &gsOperationReply);
+
+  // process 0 to N VPC states
+  int update_vpc_states(alcor::schema::GoalState &parsed_struct,
```
[the remark this answers is not in the document]
Comment (er1cthe0ne) [reply include/aca_goal_state_handler.h:32]: This is is for 0 to N VPC states, we have update_vpc_state_workitems above it to process ONE VPC state.

[the remark this answers is not in the document]
Comment (cj-chung) [reply include/aca_goal_state_handler.h:32]: What's difference between ONE VPC and 0..N VPC? Is ONE VPC is the current VPC? If it is, maybe update_current_vpc_state() is better.

**Pull request merged**